src/pages/Components/TextArea/specs.md

```diff
@@ -0,0 +1,32 @@
+---
+title: 'Specs'
+order: 3
+---
+
+## Anatomy
+
+![TextAreas specs](TextAreas-documentation.png)
+
+## Properties
+
+### CSS Tokens : layout
+
+| State   | Property      | Token            |
+| ------- | ------------- | ---------------- |
+| **all** | border-radius | `radius.m` (4px) |
+| **all** | border-width  | `border.s` (1px) |
+| **all** | padding       | `1mu` (16px)     |
+
+### CSS Tokens : state colors
+
+**You must always use the tokens, the colors are only there for your information.**
```

Comment:
You must always use the tokens, the colors are listed here for information purposes only.